lib/langchain/dependency_helper.rb

```diff
@@ -15,7 +15,12 @@ class VersionError < ScriptError; end
     # @raise [VersionError] If the gem is installed, but the version does not meet the requirements
     #
     def depends_on(gem_name, req: true)
-      gem(gem_name) # require the gem
+      if gem_name == "sqlite_vec"
```

Comment:
@CarlQLange Why is this if/else statement needed? Isn't the gem version what gets required?

Reply:
Woof, I have no memory of this. I think it was something to do with using the gem locally from something else. Sorry, I pretty much have no idea what I was doing back then!